lld/ELF/Config.h

```diff
@@ -136,6 +136,26 @@ enum LtoKind : uint8_t {UnifiedThin, UnifiedRegular, Default};
 // For -z gcs=
 enum class GcsPolicy { Implicit, Never, Always };
 
+// For -z gcs-report= and -zgcs-report-dynamic
+struct GcsReportPolicy {
```

Comment:
We don't need a struct that wraps the enum. Just define an `enum` with a free function that converts the enum to "warning"/"error".